kernel/fablib/ssh.go

```diff
@@ -135,37 +136,44 @@ func RemoteExec(sshConfig SshConfigFactory, cmd string) (string, error) {
 }
 
```

Comment:
See where RemoteExecAll is used, to see if we can consolidate on RemoteExecAllTo and rename it to RemoteExecAll